app/controllers/admin/auth_controller.rb

```diff
@@ -1,6 +1,6 @@
 class Admin::AuthController < ApplicationController
   def new
-    redirect_to '/auth/google_apps'
+    redirect_to '/auth/google_oauth2'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.